apps/judicial-system/web/src/utils/hooks/useCourtUpload/index.ts

```diff
@@ -15,8 +15,8 @@ import {
 export enum UploadState {
   ALL_UPLOADED = 'ALL_UPLOADED',
   ALL_UPLOADED_NONE_AVAILABLE = 'ALL_UPLOADED_NONE_AVAILABLE',
-  NONE_AVAILABLE = 'NONE_AVAILABLE',
-  NONE_CAN_BE_UPLOADED = 'NONE_CAN_BE_UPLOADED',
+  SOME_NOT_UPLOADED_NONE_AVAILABLE = 'NONE_AVAILABLE',
+  ALL_UPLOADED_OR_NOT_AVAILABLE = 'NONE_CAN_BE_UPLOADED',
```

Comment:
Do we not want to change the text behind =. Like this 
`  SOME_NOT_UPLOADED_NONE_AVAILABLE = 'SOME_NOT_UPLOADED_NONE_AVAILABLE',
  ALL_UPLOADED_OR_NOT_AVAILABLE = 'ALL_UPLOADED_OR_NOT_AVAILABLE',`

Reply:
Yes, absolutely.